pinot-server/src/test/java/com/linkedin/pinot/server/api/restlet/TableSizeResourceTest.java

```diff
@@ -0,0 +1,147 @@
+/**
+ * Copyright (C) 2014-2015 LinkedIn Corp. (pinot-core@linkedin.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.linkedin.pinot.server.api.restlet;
```

Comment:
Please also add new calls into tabledata manager in the OfflineTableDataManagerTest.  I suggest adding a thread that gets acquires the segments for debug purposes, and does so less often than segment replacements.